### PR TITLE
[FIX] App crashes when downloads come from WebDAV and the server is not available

### PR DESCRIPTION
**Pull request by KevLehman**
…ilable

### CODE REVIEW

File: app/file-upload/server/config/Webdav.js [hunk 1]
```diff
@@ -5,7 +5,16 @@ import { settings } from '../../../settings';
 import '../../ufs/Webdav/server.js';
 
 const get = function(file, req, res) {
-	this.store.getReadStream(file._id, file).pipe(res);
+	this.store.getReadStream(file._id, file)
+		.on('error', () => {
+			console.error('An error ocurred when fetching the file');
+			res.writeHead(400);
```
Comment (KevLehman) [inline app/file-upload/server/config/Webdav.js:11]: What can be handled better by the FE? A 500 with a message? The current 400 + log? A 404?

Comment (sampaiodiego) [reply app/file-upload/server/config/Webdav.js:11]: I'd say we should reply either 503 or 500..

Comment (KevLehman) [reply app/file-upload/server/config/Webdav.js:11]: A 503 should work  :)

Comment (KevLehman) [inline app/file-upload/server/config/Webdav.js:11]:
```suggestion
			res.writeHead(503);
```

**Review by sampaiodiego — APPROVED** [review]

**Pull request merged**